### ISSUE TITLE
HTML tags are destroyed

### ISSUE THREAD

**Issue by SventB** [thieleundklose/autotranslate#17]
I have a news record with this bodytext:

```
<p>Die Klänge der Abschlussmusik der Sterne-Zeremonie in der Hamburger Handelskammer waren noch nicht ganz verklungen, da konnten wir ein Interview mit dem Direktor der Guide-Michelin-Ausgaben für den deutschsprachigen Raum, Ralf Flinkenflügel, per Online-Schalte führen.&nbsp;</p>
<p><strong>Restaurant-Ranglisten.de:</strong> <em>Die Sterne-Zeremonie ist vor wenigen Minuten zu Ende gegangen. Mein erster Eindruck: Es gibt überraschend wenige neue Zwei-Sterne-Restaurants und Sie haben viele neue Ein-Stern-Restaurants gefunden, die zumindest ich nicht auf dem Zettel hatte. Wie ist ihr Fazit der Testsaison für den Guide 2024?</em></p>
<p><strong>Ralf Flinkenflügel</strong>: Wir sind eigentlich positiv überrascht. Viele Branchen haben hart gekämpft. Dazu gehören natürlich auch die Gastronomie. Inflation, Fachkräftemangel, schwächelnde Konjunktur, steigende Energiepreise, – dass wir wieder einen neuen Rekord an Sternen haben, ist in Anbetracht der Situation schon beachtlich.</p>
<p><strong>RR:</strong> <em>Unter den neu besternten Restaurants sind eine ganze Reihe, die erst vor wenigen Monaten eröffnet haben. In den vergangenen Jahren haben auch arrivierte Drei-Sterner den dritten Stern verloren. Ist der Michelin agiler geworden und reagiert schneller auf Entwicklungen in den Restaurants?</em></p>
<p><strong>RF:</strong> Ich denke schon, dass das der richtige Weg ist. Wenn jemand eine sehr gute Küche bietet, dann macht es einfach keinen Sinn, da zwei, drei Jahre zu warten, um unseren Leserinnen und Lesern mitzuteilen: Hier ist jemand, hier ist ein Restaurant das besondere Beachtung verdient.</p>
<p><strong>RR:</strong> <em>Schauen Sie vor der Veröffentlichung auf die Gesamtliste, ob alles gut balanciert ist, regional, oder, was Männer und Frauen oder andere Faktoren angeht?</em></p>
<p><strong>RF:</strong> Nein, unsere Kriterien richten sich nicht nach Regionen oder Listen. Es geht nach Qualität, das ist das A und O.</p>
<p><strong>RR:</strong> <em>Haben Sie bei Ihren Tests 2023 neue Trends und Entwicklungen festgestellt?</em></p>
<p><strong>RF: </strong>Nach Trends werde ich schon seit 15 Jahren gefragt. Trends entwickeln sich schleichend. Nach wie vor besteht der Trend, dass es in den gehobenen Restaurants lockerer, legerer und entspannter zugeht. Außerdem ist der Punkt der Nachhaltigkeit auch ein Faktor, der da hineinspielt.</p>
<p><strong>RR:</strong> <em>Auf das Thema Nachhaltigkeit komme ich gleich noch einmal zurück. Ich stelle aber schon fest, dass die Zahl der selbständigen Köche mit Stern zunimmt. Das hat die Folge, dass das wirtschaftliche Risiko durch die hohen Kosten, die hochwertige Gastronomie bei Produkten und Personal mit sich bringt, gestaltet werden muss. Das geht manchmal auch mit Restriktionen für den Gast einher. Ein Menü ohne Tauschmöglichkeit, feste Anfangszeiten, möglicherweise ein Ticket-System usw. Stört Sie das, oder ginge es gar nicht anders, Qualität und Wirtschaftlichkeit miteinander zu vereinen?</em></p>
<p><strong>RF:</strong> Wirtschaftlichkeit ist natürlich ganz wichtig. Dass viele, viele junge Köche den Mut haben, gerade in dieser Zeit, sich selbständig zu machen, dem muss man schon hohen Respekt zollen. Ich muss ganz ehrlich sagen, das ist schon ein gewagter Schritt. Einige gehen diesen Schritt und sind dabei auch sehr erfolgreich, wie man heute gesehen hat.</p>
<p><strong>RR: </strong><em>Wie hat sich für Sie der ‚grüne Stern‘ entwickelt? Anfangs gab es ja durchaus Kritik, weil Sie sich an mancher Stelle auf die Angaben der Gastronomen verlassen müssen und anders als bei den traditionellen Sternen nicht nur die Qualität auf dem Teller bewerten können.</em></p>
<p><strong>RF:</strong> Wir legen ziemlich strenge Maßstäbe an. Es gibt drei Schritte. Wenn wir ein Restaurant sehen, dass sich ganz besonders in dieser Richtung engagiert – wir sprechen da intern von Leuchttürmen –, bekommt es einen Fragebogen zugeschickt. Den werten wir daraufhin aus, welche Initiativen das Restaurant ergreift. Dann besuchen wir das Restaurant. Wichtig ist, dass das Personal entsprechend geschult wird und den Gästen vermittelt, was gemacht wird und welche Philosophie dahintersteckt. Schließlich führen wir ein persönliches Gespräch mit den Gastronomen, um nochmal mehr Hintergrundinformationen zu bekommen. Das ist nicht ganz so einfach, in der Kategorie aufgenommen zu werden. In den letzten zwei Jahren habe ich keine Kritik mehr an der Vorgehensweise gehört. Klar, wir können nicht kontrollieren, ob der Müll richtig getrennt wird, da muss man sich dann auch ein Stück weit auf die Ehrlichkeit der Gastronomen stützen. Aber als Gast bekommt man durchaus mit, ob ein Restaurant grün tickt, oder ob das nur eine vorgetäuschte Masche ist.</p>
<p><strong>RR: </strong><em>Ihr Inspektorenteam steht in den nächsten Monaten vor der großen Aufgabe, auch wieder in ganz Österreich Restaurants zu testen. 2025 soll der Guide erscheinen. Wie ist der Stand der Dinge? Wie läuft das organisatorisch, denn es sind ja – nach den Vergleichsmaßstäben der anderen Guides – mehrere hundert Restaurants, die getestet werden müssen?</em></p>
<p><strong>RF:</strong> Da freuen wir uns drauf! Wir sind seit einigen Wochen unterwegs. Ich war persönlich auch schon dort. Wir haben jetzt erste Eindrücke und die sind sehr positiv. Ich war in einer Region, da war ich zuletzt vor zehn Jahren und ich muss sagen: Sehr erstaunlich, was ich da erlebt habe. Aber diese Eindrücke müssen natürlich bestätigt werden. Wir haben personell nachbesetzt. Außerdem bekommen wir intensive Unterstützung von Inspektoren aus anderen Ländern, wie Frankreich, Großbritannien, oder Italien. Das klappt gut und wir sind guter Dinge.</p>
<p><strong>RR: </strong><em>Wird sich die Mehrarbeit auf die Testintensität in Deutschland auswirken? Wird es schwerer, für neue Restaurants 2025 in den Guide zu kommen?</em></p>
<p><strong>RF: </strong>Überhaupt nicht. Wir haben uns in den letzten drei, vier Jahren sehr verändert, was den internationalen Austausch von Inspektoren angeht. Es kommen Inspektoren selbst aus Japan, Südkorea, Nordamerika und aus ganz Europa. Wir tauschen uns aus. Das klappt wunderbar.</p>
<p><strong>RR: </strong><em>Wir sprechen gerade, kurz nach der Zeremonie, per Teams-App miteinander. Die Köche feiern – Sie sind nicht mit dabei. Warum?</em></p>
<p><strong>RF:</strong> Das ist nicht gewünscht, dass die Direktoren und Inspektoren in irgendeiner Art und Weise öffentlich auftreten. Klar, ich bin schon ein paar Jahre dabei, einige der Köche kennen mein Gesicht. Aber ich muss ganz ehrlich sagen, es ist mir nie angenehm, wenn ich erkannt und mit Namen angesprochen werde. Das wollen wir einfach vermeiden.</p>
```

This is translated to:

```
<p>The sounds of the closing music of the star ceremony in the Hamburg Chamber of Commerce had not yet completely faded away when we were able to conduct an interview with the director of the Guide Michelin editions for German-speaking countries, Ralf Flinkenflügel, via an online link.
<p><strong>Restaurant-Ranglisten.de:</strong> <em>The star ceremony ended a few minutes ago. My first impression: there are surprisingly few new two-star restaurants and you've found lots of new one-star restaurants that I, at least, didn't have on my radar. How would you summarise the test season for the Guide 2024?</em></p>
<p><strong>Ralf Flinkenflügel</strong>: We are actually positively surprised. Many sectors have fought hard. Of course, this also includes the catering industry. Inflation, a shortage of skilled labour, a weakening economy, rising energy prices - the fact that we have a new record number of stars is remarkable considering the situation.</p> <p
<p><strong>RR:</strong> <em>Among the newly starred restaurants are a number that only opened a few months ago. In recent years, established three-star restaurants have also lost their third star. Has the Michelin become more agile and reacts more quickly to developments in the restaurants?</em></p>
<p><strong>RF:</strong> I think that's the right way to go. If someone offers very good cuisine, then it simply doesn't make sense to wait two or three years to tell our readers: Here is someone, here is a restaurant that deserves special attention.</p
<p><strong>RR:</strong> <em>Do you look at the overall list before publication to see if everything is well balanced, regionally, or in terms of men and women or other factors?</em></p>
<p><strong>RF:</strong> No, our criteria are not based on regions or lists. It's all about quality, that's the be-all and end-all.</p>
<p><strong>RF:</strong> <em>Did you notice any new trends and developments in your 2023 tests?</em></p>
<p><strong>RF: </strong>I've been asked about trends for 15 years. Trends develop gradually. There is still a trend for upmarket restaurants to be more relaxed, casual and laid-back. Sustainability is also a factor that plays into this.</p
<p><strong>RR:</strong> <em>I'll come back to the topic of sustainability in a moment. However, I have noticed that the number of independent chefs with a Michelin star is increasing. As a result, the economic risk has to be managed due to the high costs of high-quality catering in terms of products and staff. This is sometimes accompanied by restrictions for the guest. A menu without the option to swap, fixed start times, possibly a ticket system, etc. Does this bother you, or would there be no other way to combine quality and profitability?</em></p>
<p><strong>RF:</strong> Profitability is of course very important. The fact that many, many young chefs have the courage to set up their own business, especially in these times, is something to be highly respected. I have to be honest, it's quite a bold step. Some are taking this step and are also very successful, as we have seen today.</p
<p><strong>RR: </strong><em>How has the 'green star' developed for you? Initially, there was a lot of criticism because you have to rely on the information provided by the restaurateurs in some places and, unlike with the traditional stars, you can't just rate the quality on the plate.</em></p>
<p><strong>RF:</strong> We apply fairly strict standards. There are three steps. If we see a restaurant that is particularly committed in this direction - we talk about lighthouses internally - we send it a questionnaire. We analyse it to see what initiatives the restaurant is taking. We then visit the restaurant. It is important that the staff are trained accordingly and communicate to the guests what is being done and the philosophy behind it. Finally, we talk to the restaurateurs in person to get more background information. It's not that easy to be included in the category. In the last two years, I haven't heard any criticism of the procedure. Of course, we can't check whether the waste is separated properly, so we have to rely to a certain extent on the honesty of the restaurateurs. But as a guest, you can definitely tell whether a restaurant is green or whether it's just a fake scam.</p> <p
<p><strong>RR: </strong><em>Your team of inspectors is facing the big task of testing restaurants throughout Austria again in the coming months. The guide is due to be published in 2025. How are things progressing? How is it going organisationally, as there are several hundred restaurants to be tested according to the comparative standards of the other guides?</em></p>
<p><strong>RF:</strong> We're looking forward to it! We've been travelling for a few weeks. I've already been there myself. We now have our first impressions and they are very positive. I was in a region where I was last there ten years ago and I have to say: it was amazing what I experienced there. But of course these impressions need to be confirmed. We have increased our staffing levels. We also receive intensive support from inspectors from other countries, such as France, the UK and Italy. This is working well and we are in good spirits.</p
<p><strong>RR: </strong><em>Will the extra work have an impact on the intensity of testing in Germany? Will it be more difficult for new restaurants to get into the 2025 Guide?</em></p>
<p><strong>RF: </strong>Not at all. We have changed a lot in the last three or four years in terms of the international exchange of inspectors. We have inspectors from Japan, South Korea, North America and all over Europe. We exchange ideas. That works wonderfully.</p>
<p><strong>RR: </strong><em>We are talking to each other via Teams app shortly after the ceremony. The chefs are celebrating - you're not there. Why?</em></p>
<p><strong>RF:</strong> It's not desirable for the directors and inspectors to appear in public in any way. Sure, I've been there for a few years, some of the chefs know my face. But I have to be honest, I'm never comfortable being recognised and called by name. We simply want to avoid that.</p>
```

There are errors in HTML structure now:

```
</p> <p
<p><strong>RR:</strong> <em>Among the newly
```

or missing closing tags:

```
via an online link.
<p><strong>
```

Translating this text directly on deepl.com doesn't result in this.

**Comment by mikezimmer-tuk** [comment]
Hi Sven, thx for your Feedback.

I think this Bug is fixed since last version v1.0.3. Changing Tag Handling to HTML fixed it.

Your Example Content should be translated fine now:

`<p>The sounds of the closing music of the star ceremony in the Hamburg Chamber of Commerce had not yet completely faded when we were able to conduct an interview with the director of the Guide Michelin editions for German-speaking countries, Ralf Flinkenflügel, via an online link.</p>

<p><strong>Restaurant-Ranglisten.de:</strong> <em>The star ceremony ended a few minutes ago. My first impression: there are surprisingly few new two-star restaurants and you have found many new one-star restaurants that at least I didn't have on my radar. How would you summarise the test season for Guide 2024?</em></p>

<p><strong>Ralf Flinkenflügel</strong>: We are actually positively surprised. Many sectors have fought hard. Of course, this also includes the catering industry. Inflation, a shortage of skilled labour, a weakening economy, rising energy prices - the fact that we have a new record number of stars is remarkable considering the situation.</p>

<p><strong>RR</strong>: <em>Among the newly starred restaurants are a number that only opened a few months ago. In recent years, established three-star restaurants have also lost their third star. Has Michelin become more agile and reacts more quickly to developments in the restaurants?</em></p>

<p><strong>RF</strong>: I think that's the right way to go. If someone offers very good cuisine, then it simply doesn't make sense to wait two or three years to tell our readers: Here's someone, here's a restaurant that deserves special attention.</p>

<p><strong>RR</strong>: Do <em>you look at the overall list before publication to see whether everything is well balanced, regionally, or in terms of men and women or other factors?</em></p>

<p><strong>RF:</strong> No, our criteria are not based on regions or lists. It's all about quality, that's the be-all and end-all.</p>

<p><strong>RR</strong>: <em>Have you identified any new trends and developments in your 2023 tests?</em></p>

<p><strong>RF </strong>: People have been asking me about trends for 15 years. Trends develop gradually. There is still a trend for upscale restaurants to be more relaxed, casual and laid-back. Sustainability is also a factor that plays into this.</p>

<p><strong>RR</strong>: <em>I'll come back to the subject of sustainability in a moment. However, I have noticed that the number of independent chefs with a Michelin star is increasing. As a result, the economic risk has to be managed due to the high costs of high-quality catering in terms of products and staff. This is sometimes accompanied by restrictions for the guest. A menu without the option of swapping, fixed start times, possibly a ticket system, etc. Does that bother you, or is there no other way to combine quality and efficiency?</em></p>

<p><strong>RF</strong>: Profitability is of course very important. The fact that many, many young chefs have the courage to set up their own business, especially in these times, is something to be highly respected. I have to be honest, it's a bold step. Some take this step and are very successful, as we have seen today.</p>

<p><strong>RR </strong>:<em>How did the 'green star' develop for you? There was some criticism at first because you have to rely on the information provided by the restaurateurs in some places and, unlike the traditional stars, you can't just assess the quality on the plate.</em></p>

<p><strong>RF:</strong> We apply fairly strict standards. There are three steps. If we see a restaurant that is particularly committed in this direction - we talk about lighthouses internally - we send it a questionnaire. We analyse it to see what initiatives the restaurant is taking. We then visit the restaurant. It is important that the staff are trained accordingly and communicate to the guests what is being done and the philosophy behind it. Finally, we talk to the restaurateurs in person to get more background information. It's not that easy to be included in the category. In the last two years, I haven't heard any criticism of the procedure. Of course, we can't check whether the waste is separated properly, so we have to rely to a certain extent on the honesty of the restaurateurs. But as a guest, you can definitely tell whether a restaurant is green or whether it's just a fake scam.</p>

<p><strong>RR </strong>: Over the<em>next few months, your team of inspectors will be faced with the major task of testing restaurants throughout Austria again. The guide is due to be published in 2025. How are things progressing? How is it going organisationally, as there are several hundred restaurants to be tested - according to the comparative standards of the other guides?</em></p>

<p><strong>RF</strong>: We're looking forward to it! We've been travelling for a few weeks now. I've already been there myself. We now have our first impressions and they are very positive. I was in a region where I was last there ten years ago and I have to say: it was amazing what I experienced there. But of course these impressions need to be confirmed. We have increased our staffing levels. We also receive intensive support from inspectors from other countries, such as France, the UK and Italy. This is working well and we are in good spirits.</p>

<p><strong>RR </strong>:<em>Will the extra work have an impact on the intensity of testing in Germany? Will it be harder for new restaurants to get into the 2025 Guide?</em></p>

<p><strong>RF </strong>: Not at all. We have changed a lot in the last three or four years in terms of the international exchange of inspectors. We have inspectors from Japan, South Korea, North America and all over Europe. We exchange ideas. It works wonderfully.</p>

<p><strong>RR </strong>:<em>We are talking to each other via Teams app shortly after the ceremony. The chefs are celebrating - you're not there. Why is that?</em></p>

<p><strong>RF</strong>: It's not desirable for the directors and inspectors to appear in public in any way. Sure, I've been there for a few years, some of the chefs know my face. But I have to be honest, I never feel comfortable being recognised and called by name. We simply want to avoid that.</p>
`

Because of this i will close that Issue now.

best regards - Mike

**Comment by SventB** [comment]
Ok, thank you
